Xamarin.Forms.Platform.WinRT/DatePickerRenderer.cs

```diff
@@ -96,7 +96,19 @@ void UpdateMaximumDate()
 		void UpdateMinimumDate()
 		{
 			DateTime mindate = Element.MinimumDate;
-			Control.MinYear = new DateTimeOffset(mindate);
+
+			try
+			{
+				Control.MinYear = new DateTimeOffset(mindate);
+			}
+			catch(ArgumentOutOfRangeException)
+			{
+				// This will be thrown if you try to set MinYear to DateTime.MinValue when the UTC offset is positive
+				// because the time is out range once it's converted. In that case, specify the Kind as UTC 
+				// so it doesn't get converted.
+				mindate = DateTime.SpecifyKind(mindate, DateTimeKind.Utc);
```

Comment:
I'm sure there's a reason, but why don't we just always get the Date value and use that instead of adding a `try catch` and additional logic?